health/micro-ui/web/micro-ui-internals/packages/modules/health-hrms/src/components/Utils/index.js

```
@@ -0,0 +1,56 @@
+/* methid to get date from epoch */
+export const convertEpochToDate = (dateEpoch) => {
+  // Returning null in else case because new Date(null) returns initial date from calender
+  if (dateEpoch) {
+    const dateFromApi = new Date(dateEpoch);
+    let month = dateFromApi.getMonth() + 1;
+    let day = dateFromApi.getDate();
+    let year = dateFromApi.getFullYear();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'year' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)